### PR TITLE
Set timezone to UTC

### LINKED ISSUE #1560: Change timezone to UTC
Currently, the timezone in olympia is "America/Los_Angeles" which creates issues with displaying dates, for example the wait time for add-ons in the review queue.

Most of the code is not timezone aware, some small pieces are. There is even some timezone translation for ES going on, even though `USE_TZ` is _false_. The prod server itself runs on UTC.

So, let's change the timezone to UTC?!

### PR DESCRIPTION
**Pull request by wagnerand**
Fix #1560

### CODE REVIEW

**Comment by wagnerand** [comment]
r?

File: apps/addons/tests/test_views.py [hunk 1]
```diff
@@ -1504,8 +1505,8 @@ def test_button_caching(self):
 
         response = client.get(url_with_build, follow=True)
         fmt = '%a, %d %b %Y %H:%M:%S GMT'
-        expires = datetime.strptime(response['Expires'], fmt)
-        assert (expires - datetime.now()).days >= 365
+        expires = datetime.datetime.strptime(response['Expires'], fmt)
+        assert (expires - now).days >= 365
```
Comment (diox) [inline apps/addons/tests/test_views.py]: Even simpler:
```diff
 -        fmt = '%a, %d %b %Y %H:%M:%S GMT'
 -        expires = datetime.strptime(response['Expires'], fmt)
 -        assert (expires - datetime.now()).days >= 365
 +        self.assertCloseToNow(response['Expires'],
 +                              now=datetime.now() + timedelta(days=365))
```

File: apps/amo/tests/test_views.py [hunk 2]
```diff
@@ -283,8 +284,8 @@ def test_jsi18n_caching(self):
 
         response = client.get(url_with_build, follow=True)
         fmt = '%a, %d %b %Y %H:%M:%S GMT'
-        expires = datetime.strptime(response['Expires'], fmt)
-        assert (expires - datetime.now()).days >= 7
+        expires = datetime.datetime.strptime(response['Expires'], fmt)
+        assert (expires - now).days >= 7
```
Comment (diox) [inline apps/amo/tests/test_views.py]: See above, lets use `assertCloseToNow`

**Comment by diox** [comment]
r+wc 

Let's wait till after the tag to merge this.

**Pull request merged**